firestore/integration_test_internal/src/android/promise_android_test.cc

```diff
@@ -195,12 +195,12 @@ class TestCompletion : public TestCompletionBase<PublicType, InternalType> {
     if (result == nullptr) {
       result_.reset(nullptr);
     } else {
-      result_ = MakeUnique<PublicType>(*result);
+      result_ = std::make_unique<PublicType>(*result);
```

Comment:
Like you noticed, we still can't use `std::make_unique` (there's `absl::make_unique`, but it's not available on Android). It would be unfortunate to avoid using `std::unique_ptr` because of this, though. Consider writing a custom version of `make_unique` that returns `std::unique_ptr` instead (the implementation can likely be mostly copied from `MakeUnique`). Alternatively, you could directly call the `unique_ptr` constructor and `new` the object.